components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -218,6 +220,7 @@ export const AudioPlayer = ({
             {!!embeddedPlayerUrl && (
               <IconButton
                 className={classes.embedBtn}
+                aria-label="Embed Code"
```

Comment:
This should probably update based on player state: `aria-label={embedCodeShown ? 'Hide embed code' : 'Show embed code'}`

Reply:
Fixed!